src/Model/SerieManager.php

```diff
@@ -83,16 +83,40 @@ public function upload()
         return $filePath;
     }
 
+    /**
+     * @param array $data
+     * @return string
+     */
     public function insert(array $data)
     {
         try{
             $data['link_picture'] = $this->upload();
```

Comment:
c'est une solution. Mais plutot que de créer une méthode insert dans le SerieManager, tu pourrais seulement faire le 
 `$data['link_picture'] = $this->upload(); ` dans le controller juste avant d'appeler la méthode insert($data)